Error when trying to build

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 4.673 s
[INFO] Finished at: 2020-03-19T18:41:01-05:00
[INFO] Final Memory: 12M/303M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project warc-parser: Could not resolve dependencies for project com.searchmetrics.ds:warc-parser:jar:0.0.1-SNAPSHOT: Failed to collect dependencies at org.slf4j:slf4j-api:jar:1.7.21: Failed to read artifact descriptor for org.slf4j:slf4j-api:jar:1.7.21: Could not transfer artifact org.slf4j:slf4j-api:pom:1.7.21 from/to central (http://repo1.maven.org/maven2): Failed to transfer file: http://repo1.maven.org/maven2/org/slf4j/slf4j-api/1.7.21/slf4j-api-1.7.21.pom. Return code is: 501 , ReasonPhrase:HTTPS Required. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
mv: cannot stat 'target/warc-parser-*.jar': No such file or directory

I am getting this error when I try to build it. I have Maven and Java installed correctly.

--- a/pom.xml
+++ b/pom.xml
@@ -14,7 +14,7 @@
   <repositories>
     <repository>
       <id>central</id>
-      <url>http://repo1.maven.org/maven2</url>
+      <url>https://repo1.maven.org/maven2</url>
     </repository>
     <repository>
       <id>bintray</id>

